File: projects/neptune/src/main/java/io/fairspace/neptune/config/OidcProperties.java
package io.fairspace.neptune.config;

import io.fairspace.oidc_auth.config.OidcConfig;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

@ConfigurationProperties(prefix = "security.oidc")
@Configuration
public class OidcProperties extends OidcConfig {
}

File: projects/ceres/src/main/kotlin/io/fairspace/ceres/config/OidcProperties.kt
package io.fairspace.ceres.config

import io.fairspace.oidc_auth.config.OidcConfig
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@ConfigurationProperties(prefix = "security.oidc")
@Configuration
class OidcProperties : OidcConfig()
